action-server/actions/lib/environment.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+
+def get_env(name):
+    value = os.environ.get(name)
```

Comment:
FYI using `os.environ[name]` already gives an error when the env variable is not defined

Reply:
will fix that in next PR